browser_tests/fixtures/ComfyPage.ts

```diff
@@ -338,6 +340,99 @@ export class ComfyPage {
     return `./browser_tests/assets/${fileName}`
   }
 
+  async fitToView(
+    options: {
+      selectionOnly?: boolean
+      zoom?: number
+      padding?: number
+    } = {}
+  ) {
```

Comment:
This method is a lot of logic to put in a fixture.
Normally I like fixtures to be as close to describing what a user would do as possible.

The general rule of browser tests is: unless you expect a user to open the console and type in a series of commands, the test shouldn't do that either.